skull-king-app/src/components/PlayerStatusCard/PlayerStatusCard.less

```diff
@@ -180,6 +180,184 @@
+    border-radius: 12px;
+    border: 2px solid @enabledButtonColor;
+    background-color: rgba(221, 221, 255, 0.1);
+    color: inherit;
+    font-family: inherit;
+    font-weight: 600;
+    
+    &:disabled {
+      border-color: @disabledButtonColor;
+      background-color: rgba(204, 204, 204, 0.1);
+      color: @disabledButtonColor;
+    }
+    
+    &:focus,
+    &:active,
+    &.show {
+      border-color: @activePlayerColor !important;
+      box-shadow: 0 0 0 0.2rem rgba(136, 136, 221, 0.25) !important;
+      background-color: rgba(221, 221, 255, 0.2) !important;
+      color: inherit !important;
+    }
+    
+    &:hover:not(:disabled) {
+      background-color: rgba(221, 221, 255, 0.15);
+      border-color: @activePlayerColor;
+    }
+  }
+
+  .bonusDropdownMenu {
+    border-radius: 12px;
+    border: 2px solid @enabledButtonColor;
+    background-color: rgba(221, 221, 255, 0.95);
+    backdrop-filter: blur(4px);
+    max-height: 300px;
+    overflow-y: auto;
+    
+    .bonusDropdownItem {
+      color: inherit;
+      font-weight: 600;
+      padding: 8px 12px;
+      
+      &:hover:not(:disabled) {
+        background-color: rgba(136, 136, 221, 0.2);
+        color: inherit;
+      }
+      
+      &:disabled {
+        color: @disabledButtonColor;
+      }
+    }
+    
+    .bonusDropdownCloseItem {
+      color: @activePlayerColor;
+      font-weight: bold;
+      border-top: 1px solid rgba(136, 136, 221, 0.3);
+      
+      &:hover {
+        background-color: rgba(136, 136, 221, 0.2);
+        color: @activePlayerColor;
+      }
+    }
+  }
+
+  .selectedBonusesDisplay {
+    width: 100%;
+
+    .bonusListContainer {
+      background-color: rgba(221, 221, 255, 0.15);
+      border: 2px solid @enabledButtonColor;
+      border-radius: 12px;
+      padding: 12px;
+      font-size: 0.9em;
+
+      .bonusHeader {
+        font-family: "Jim Nightshade", system-ui;
+        font-weight: 400;
+        font-size: 1.1em;
+        margin-bottom: 8px;
+        color: @activePlayerColor;
+        text-align: center;
+      }
+
+      .selectedBonusItem {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        padding: 6px 8px;
+        margin: 4px 0;
+        background-color: rgba(255, 255, 255, 0.3);
+        border-radius: 8px;
+        border: 1px solid rgba(170, 170, 255, 0.3);
+        font-weight: 600;
+
+        &:last-child {
+          border-bottom: 1px solid rgba(170, 170, 255, 0.3);
+        }
+
+        .removeBonusButton {
+          background: none;
+          border: 2px solid @enabledButtonColor;
+          color: @enabledButtonColor;
+          font-size: 14px;
+          font-weight: bold;
+          cursor: pointer;
+          padding: 0;
+          width: 24px;
+          height: 24px;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          border-radius: 50%;
+          transition: all 0.2s ease;
```

Comment:
Let's just change this to a big red 'x' in the normal UI font Remove the border, the highlight, etc.